web-24wi/assignments/harleehair/frontend/week1/css/style.css

```diff
@@ -1,172 +1,181 @@
-*{
+* {
     box-sizing: border-box;
-}
-
-body{
+  }
+  
+  body {
     margin: 0;
     padding: 0;
     background-color: #efefef;
     font-family: 'Open Sans', sans-serif;
-}
-
-.main-nav{
+  }
+  
+  .page {
+    display: grid;
+    row-gap: 30px;
+    column-gap: 30px;
+    width: 100%;
+    padding: 30px;
+    margin: 0 auto;
+    grid-template-areas:
+      "order payment"
+      "total total";
+    grid-template-columns: 1fr 1fr;;
+  }
+  
+  .main-nav {
     background-color: #43281c;
     color: #fbf2c0;
     font-size: 16px;
     display: flex;
     align-items: center;
     justify-content: center;
     font-weight: normal;
-}
-
-.logo{
+  }
+  
+  .logo {
     height: 25px;
     position: relative;
     top: 3px;
-}
-
-.page{
-    display: grid;
-    row-gap: 30px;
-    column-gap: 30px;
-    width: 100%;
-    padding: 30px;
-    margin: 0 auto;
-    grid-template-areas: "order payment" "total total";
-    grid-template-columns: 1fr 1fr;;
-}
-
-.box{
+  }
+  
+  .box {
     background-color: #eee1d5;
     border-radius: 5px;
-    box-shadow: 0 1px 3px rgb(0, 0, 0 / 0.2);
+    box-shadow: 0 1px 3px rgb(0 0 0 / 0.2);
```

Comment:
nice, I'm learning this new CSS syntax. It looks like the `0.2` means mostly transparent.
https://css-tricks.com/no-comma-color-functions-in-css/